doc/development.md

```diff
@@ -26,7 +26,7 @@ Clone your fork:
 
 Install additional packages for Fedora, CentOS, and RHEL:
 
-    sudo dnf install -y `cat ci/check-patch.packages`
+    ci/install-pkg.sh
```

Comment:
This is not used in the ci - it is used only for preparing development environment manually, so why does it belong in the ci directory?

Reply:
right, moved to `contrib`, it should pertain there better.